tests/SIL.LCModel.Tests/DomainServices/WritingSystemServicesTests.cs

```diff
@@ -255,6 +255,34 @@ public void UpdateWritingSystemListField_RemovesMergedCodeAfterMergeWith()
 
 		/// <summary/>
 		[Test]
+		public void DeleteWritingSystem()
+		{
+			int m_wsEn = Cache.WritingSystemFactory.GetWsFromStr("en");
+			int m_wsFr = Cache.WritingSystemFactory.GetWsFromStr("fr");
+
+			CoreWritingSystemDefinition enBlz;
+			WritingSystemServices.FindOrCreateWritingSystem(Cache, null, "blz", false, false, out enBlz);
+
+			var revIndex = Cache.ServiceLocator.GetInstance<IReversalIndexRepository>().FindOrCreateIndexForWs(m_wsEn);
+
+			var entry1 = SenseOrEntryTests.CreateInterestingLexEntry(Cache);
+			var msa1 = Cache.ServiceLocator.GetInstance<IMoStemMsaFactory>().Create();
+			entry1.MorphoSyntaxAnalysesOC.Add(msa1);
```

Comment:
I think this and the next line are extra

Reply:
Yes, I removed extra information.